docs/05-internals/design/api-v2/gravsearch.md

```diff
@@ -295,23 +295,10 @@ on Gravsearch). This is implemented as follows:
 When the non-triplestore-specific version of a SPARQL query is generated, statements that do not need
 inference are marked with the virtual named graph `<http://www.knora.org/explicit>`.
 
-When the triplestore-specific version of the query is generated:
-
-- If the triplestore is GraphDB, `SparqlTransformer.transformKnoraExplicitToGraphDBExplicit` changes statements
-  with the virtual graph `<http://www.knora.org/explicit>` so that they are marked with the GraphDB-specific graph
-  `<http://www.ontotext.com/explicit>`, and leaves other statements unchanged. 
-  `SparqlTransformer.transformKnoraExplicitToGraphDBExplicit` also adds the `valueHasString` statements which GraphDB needs 
-  for text searches.
-
-- If Knora is not using the triplestore's inference (e.g. with Fuseki),
-  `SparqlTransformer.transformStatementInWhereForNoInference` removes `<http://www.knora.org/explicit>`, and expands unmarked
-  statements using `rdfs:subClassOf*` and `rdfs:subPropertyOf*`.
-
-Gravsearch also provides some virtual properties, which take advantage of forward-chaining inference
-as an optimisation if the triplestore provides it. For example, the virtual property
-`knora-api:standoffTagHasStartAncestor` is equivalent to `knora-base:standoffTagHasStartParent*`. If Knora is not using the triplestore's inference, `SparqlTransformer.transformStatementInWhereForNoInference`
+When the triplestore-specific version of the query is generated, this could make use of a triplestore's inference. Currently, no triplestore-based inference is used because of poor performance. Instead, the API expands the prequery on basis of the available ontologies, to achieve the same results as if inference was used. For that reason, `SparqlTransformer.transformStatementInWhereForNoInference` removes `<http://www.knora.org/explicit>`, and expands unmarked statements using `UNION` statements for all subclasses and subproperties (equivalent to `rdfs:subClassOf*` and `rdfs:subPropertyOf*`). Similarly, `SparqlTransformer.transformStatementInWhereForNoInference`
```

Comment:
This is a bit misleading for me (maybe not for the user anymore ;) because the fact is we don't offer other triplestores support anymore, so there are. no triplestore-specific queries anymore too.

Reply:
Changed it.